<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NABaseInfoDialog</class>
 <widget class="QDialog" name="NABaseInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>账户基本信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>账户代码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtCode"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>账户文件名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="edtFileName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>账户简称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtSName"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>使用的科目系统</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cmbSubSys"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>账户全称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="edtLName"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>起账时间</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="edtStartDate">
       <property name="displayFormat">
        <string>yyyy-M</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>NABaseInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
